docs/core/howto/constants.rst

```diff
@@ -1,7 +0,0 @@
-Symbolic Constants
```

Comment:
I think that we should keep this documentation page, but mark the page as `:orphan:` and remove it from the index.

There might be links to this page and if we remove it they will get 404 ... if we keep them it will contain the info to the migrated project

Reply:
I tried to remove any mention of the deprecated module. Im not very familiar with the documentation syntax.

I can revert this removal and look into the `orphan`